viam-cartographer.go

```diff
@@ -930,7 +946,7 @@ func (slamSvc *builtIn) getAndSaveDataDense(ctx context.Context, cams []camera.C
 
 	var fileType string
 	switch slamSvc.slamMode {
-	case slam.Dim2d:
+	case slam.Dim2d, slam.Dim3d:
```

Comment:
Should we throw an error here with "Dim3d is not implemented" like we do in CreateTimestampFile?

Reply:
Yes. Thank you for pointing this out.